test/validation/api/traffic_mngr/traffic_mngr.c

```diff
@@ -2802,30 +3012,35 @@ static int test_sched_node_priority(const char *shaper_name,
 				    const char *node_name,
 				    uint32_t    num_pkts)
 {
+	uint32_t pkts_in_order, base_idx, queue_idx, idx, unique_id;
+	uint32_t found_queues, num_queues, pkt_cnt;
+	uint32_t pkt_len, base_pkt_len, total_pkt_cnt, pkts_sent;
+	tm_queue_desc_t *queues_set[MAX_TM_PRIORITIES];
 	odp_tm_queue_t *tm_queues, tm_queue;
+	tm_queue_desc_t *queue_array;
 	tm_node_desc_t *node_desc;
-	queue_array_t  *queue_array;
-	pkt_info_t      pkt_info;
-	uint32_t        total_num_queues, max_queues, num_queues, pkt_cnt;
-	uint32_t        pkts_in_order, base_idx, queue_idx, idx, unique_id;
-	uint32_t        pkt_len, base_pkt_len, total_pkt_cnt, pkts_sent;
-	int             priority;
+	pkt_info_t pkt_info;
+	int priority;
+	uint16_t sz;
```

Comment:
There are unnecessary formatting and ordering changes here. Only one existing line needs to be modified and two new lines (for queues_set and sz definitions) need to be added.